### ISSUE TITLE
Duplicated copies of text with one character less

### ISSUE THREAD

**Issue by bincdianshangcun** [xrelkd/clipcat#360]
Hello there, thanks for building this useful/fast clipboard master. 
I did a fresh installation, and generated default configuration.

Version: `clipcat-0.16.4-1.el7.x86_64.rpm`
OS: `Fedora 39 (i3wm spin)`
Linux: `Linux hostname 6.6.9-200.fc39.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Jan  1 20:05:54 UTC 2024 x86_64 GNU/Linux`

I copied below one line of text:
`clipcat-notify : A tool for watching clipboard event`

Then I run `clipcat list`, I got:
```
aafc9fad4a9e8e31: clipcat-notify : A tool for watching clipboard event
82d1ecb53a7bae45: clipcat-notify : A tool for watching clipboard even
f1bb98bc9c2a069d: clipcat-notify : A tool for watching clipboard eve
585143aee4ebadd4: clipcat-notify : A tool for watching clipboard ev
de29b58e0f8e8d60: clipcat-notify : A tool for watching clipboard e
fa786c29ded0b62d: clipcat-notify : A tool for watching clipboard 
ee28805046883816: clipcat-notify : A tool for watching clipboard
afa9d04a9803fc45: clipcat-notify : A tool for watching clipboar
7e4c1ff57255fd8e: clipcat-notify : A tool for watching clipboa
ddb6ea8d7639d530: clipcat-notify : A tool for watching clipbo
e7611dd5e24ef712: clipcat-notify : A tool for watching clipb
a1d2187dcde34745: clipcat-notify : A tool for watching clip
63dd56d6d1a35aeb: clipcat-notify : A tool for watching cli
c9f062bba039fe3d: clipcat-notify : A tool for watching cl
c28dfa62a1f16dd9: clipcat-notify : A tool for watching c
0a9812fe83df375f: clipcat-notify : A tool for watching 
e31c2d98e0a52eb7: clipcat-notify : A tool for watching
eb2bf0c5c51c11cc: clipcat-notify : A tool for watchin
a1810f4c8abc2882: clipcat-notify : A tool for watch
a374404921d80024: clipcat-notify : A tool for watc
d30ef270112841d4: clipcat-notify : A tool for wat
68b37f2313e8aa66: clipcat-notify : A tool for wa
ddf84d23cf00658b: clipcat-notify : A tool for w
d92230f1c07aa22a: clipcat-notify : A tool for 
6480806c14063a0f: clipcat-notify : A tool fo
39b21deb322f8025: clipcat-notify : A tool f
5f83789bcf701c95: clipcat-notify : A tool
6ce7f3c305eef1d3: clipcat-notify : A to
34f0de43ac103efc: clipcat-notify : A t
030157650b46b955: clipcat-notify : A 
92ce5f838735a449: clipcat-notify : 
48db08401c7c63e2: clipcat-notify 
1aa6eac02319d6e1: clipcat-notify
4cd4a3fa55cd6f65: clipcat-not
b2edc3d3fe264205: clipcat-no
5bdd01c9691c19bf: clipcat-n
b76b3799e52d0544: clipcat-
bfa6237aaa3dc008: cl
```

**Comment by workflow** [comment]
This is likely because you have `synchronize_selection_with_clipboard = true` in your `clipcatd.toml`, so every selection you make will get copied.

That being said, it would be super helpful to have a setting to debounce that, i.e. copy selection only once a second or so, or automatically clean up the subsets.

**Comment by workflow** [comment]
Looks like a duplicate of https://github.com/xrelkd/clipcat/issues/4

**Comment by ivan94fi** [comment]
Hi, this is still relevant. Is there some option to reduce the cluttering of history? An example of this behavior is the following (selecting some text, part of  `ls --help` output, from vscode integrated terminal):
```text
a77a0d51ad4cdb24: with -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
27f4924a5e91f4ec: ith -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
1d099379b13e8873: th -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
1d03c27e02b314cc: h -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
0e38ca5bc5a6969a:  -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
0b72b86433c3252b: -l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
e0e81cd1db99b175: l, print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
b3e1639e9276a982: , print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
42dd2ee2c443e99c:  print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
d18c98798c9e481c: print the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
2d75b34cd269f636: rint the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
d02794f6d37ff888: int the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
31752e6d5a99dbe0: nt the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
9e2e2ccb0e36b2cf: t the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
ce5f33d0f810cc5d: he author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
eff98aec7afdc9a3: the author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
a72566e93db2b713: uthor of each file\n  -b, --escape               print C-style escapes for nongraphic characters
673dd52309d08f6b:  author of each file\n  -b, --escape               print C-style escapes for nongraphic characters
546ebd266887a94e: r of each file\n  -b, --escape               print C-style escapes for nongraphic characters
3ac9dba4287ee438: ach file\n  -b, --escape               print C-style escapes for nongraphic characters
9e71bf98b5fb54a5: aphic characters
c1f7ba89fe615767: phic characters
4b5c4e345212a892: ic characters
186e8489748b3f41: c characters
941bc54ea9d55b6a:  characters
714d07e749ddcfbe: haracters
```
